__tests__/app.test.js

```diff
@@ -0,0 +1,38 @@
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data");
+const e = require("express");
```

Comment:
I am really not a big fan of this variable name being e. This variable name doesnt tell me anything at all about what it actually is supposed to represent. I also dont believe this needs to be required in this file either.